front/lib/api/assistant/actions/retrieval.ts

```diff
@@ -461,7 +461,7 @@ export async function* runRetrieval(
     }
   }
 
-  const model = configuration.generation?.model;
```

Comment:
Let's make sure there is no code around that that handled the case model is null